docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -148,3 +148,10 @@ You may notice some differences in the coverage metrics reported by the old and
 
 -   The old Coverage engine ignores any files on a coverage report that aren't present on the repository on that given commit.
 -   The old Coverage engine may ignore additional files since it shares [ignore rules](../../repositories-configure/ignoring-files.md) with the Codacy analysis engine.
+
+## Differences in coverage metrics between the Coverage and Quality pull request pages
+
+You may notice some differences in the coverage metrics reported by the [Coverage Pull requests page](../../repositories-coverage/pull-requests.md) and [Quality Pull requests page](../../repositories/pull-requests.md). This may happen due to a number of reasons:
+
+-   The latest commit to the pull request was pushed before the new Coverage engine was enabled. To resolve this, please re-analyze the pull request by pushing an empty commit.
```

Comment:
Is this the only way to reanalyze? No UI interface yet?

Reply:
Yes; you can help the reader also by showing how to do an empty commit:

`git commit --allow-empty -m "Trigger analysis"`